aws/sqs/check.go

```
@@ -0,0 +1,46 @@
+package sqs
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/AirHelp/sensu-plugins-go/aws/cloudwatch"
+	"github.com/AirHelp/sensu-plugins-go/sensu"
+)
+
+func Check(cfg *Configuration) {
+	stats := NewStats(cfg.TypeOfCheck)
+
+	queues, err := collectDataAboutQueues(cfg)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(sensu.UnknownExitCode)
```

It's just a service. I would return an error in such a case and allow the caller to decide what to do in such a case. In this particular case `main.go` should decide if app should be closed or not.

---

fixed here: https://github.com/AirHelp/sensu-plugins-go/pull/11/commits/b4c8960f7a183e322c108ec5056a94f7de1f133b#diff-3cf502e3fe34f34252eb1c3741faf402R41-R46